docs/guides/compose.md

```diff
@@ -0,0 +1,260 @@
+<!--
+title: Serverless Framework - Composing services
+menuText: Composing services
+menuOrder: 3
+layout: Doc
+-->
+
+<!-- DOCS-SITE-LINK:START automatically generated  -->
+
+### [Read this on the main serverless docs site](https://www.serverless.com/framework/docs/guides/compose/)
+
+<!-- DOCS-SITE-LINK:END -->
+
+# Composing Serverless Framework services
+
+Deploying multiple services in a monorepository is a very common pattern across larger teams. Serverless Framework Compose is meant to simplify deploying and orchestrating multiple services:
+
+- Deploy multiple services in parallel
+- Deploy dependent services in a specific order
```

Comment:
Should it just say `Deploy services in a specific order`?

Reply:
👍  done